package.json

```diff
@@ -24,15 +24,22 @@
     ],
     "permissionSets": [
       {
-        "permissionName": "ui-plugin-bursar-export.bursar-exports.all",
-        "displayName": "Bursar exports: Bursar admin",
+        "permissionName": "ui-plugin-bursar-export.bursar-exports.view",
```

Comment:
Could you please investigate should we add displayName and translation for this permission?